exporter/metric/stdout/stdout.go

```diff
@@ -0,0 +1,138 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package stdout
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"go.opentelemetry.io/api/core"
+	"go.opentelemetry.io/sdk/export"
+	"go.opentelemetry.io/sdk/metric/aggregator"
+)
+
+type Exporter struct {
+	options Options
+}
+
+// Options are the options to be used when initializing a stdout export.
+type Options struct {
+	// File is the destination.  If not set, os.Stdout is used.
+	File *os.File
+
+	// PrettyPrint will pretty the json representation of the span,
+	// making it print "pretty". Default is false.
+	PrettyPrint bool
+
+	// Quantiles are the desired aggregation quantiles for measure
+	// metric data, used when the configured aggregator supports
+	// quantiles.
+	Quantiles []float64
```

Comment:
Do we believe quantiles should be an attribute per-metric, or a property of the exporter as a whole and not configurable per-metric?

Reply:
I've prepared this stdout exporter as a demonstration. For the demonstration, I think having a single configured default for quantiles is sensible. I believe we (the industry) should have a configuration protocol for configuring per-metric quantiles, per-metric aggregation dimensions, and per-metric aggregations.  The task would be to define a protocol that could be used from a `.yaml`, or returned by a server when a client connects, etc.

Reply:
Great. Maybe leave a comment to that effect? Otherwise I worry people writing exporters will just copy the stdout exporter and its config settings too...

Reply:
Done. I'll be supervising construction of statsd, prometheus, and openmetrics exporters. I don't think a common configuration mechanism will emerge, but I would like if OTel moved in that direction--that configuration is loaded from the server to configure which metrics to export, which dimensions, which export types (e.g., summary vs histogram), which quantiles / buckets, etc.

Presently, "which aggregation" is controlled by the aggregation selector, "which dimensions" are controlled by the batcher (currently the instrument's recommended labels), and "which export type" is controlled by the exporter directly.